### PR TITLE
JEI hide category and IServer change

### PR DESCRIPTION
**Pull request by kindlich**

### CODE REVIEW

File: docs/Contribute/OnlineEditor_Create.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+- Entries start with a `-`
+- Then comes the (shown, English) name for the group or entry, followed by a `:`
+- If you are creating a grouping (e.g. `Vanilla` or `Mods`) proceed on the next line, with two spaces inlined.
+- If you are creating an actual reference to a page file, add it on the same line, after the `:` and a space. Make sure to wrap it in single quotes `'` to ensure that the build works as expected. The path is relative to the `docs` folder, so `docs/Vanilla/Commands.md` becomes `Vanilla/Commands.md`.
+
+For examples check the [current mkdocs.yml file on github](https://github.com/CraftTweaker/CraftTweaker-Documentation/blob/master/mkdocs.yml). Alternatively, edit this file and add an own example here.
+
+## Save/Commit the changes
+*Note: This description originates from the edit files guide but the same principles apply, feel free to substitute with an own version if needed*
```
Comment (jaredlll08) [inline docs/Contribute/OnlineEditor_Create.md]: "with an own" -> "with your own"

File: docs/Contribute/OnlineEditor_Edit.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+You may want to edit the page because you have found a typo or want to include a better example, whatever the reason, feel free to commit changes and eventually [file a Pull Request](/Contribute/PullRequest).  
+
+However, remember that you can only edit the English version, translations are handled via [CrowdIn](https://crowdin.com/project/crafttweaker-documentation/).
+
+
+## Locate the file
+If you want to edit an existing file, you first need to locate it.  
+
+
+In most cases you can find the file by checking its Wiki Url.
```
Comment (jaredlll08) [inline docs/Contribute/OnlineEditor_Edit.md]: "its" -> "it's" maybe? I'm not sure on this one

File: docs/Contribute/OnlineEditor_Edit.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+## Locate the file
+If you want to edit an existing file, you first need to locate it.  
+
+
+In most cases you can find the file by checking its Wiki Url.  
+
+Rule of thumb: The `#` is the docs folder, everything after that are nested folders and eventually the file.  
+
+For example, the file creating  
+<http://docs.blamejared.com/en/#AdvancedFunctions/Arrays_and_Loops/>
```
Comment (jaredlll08) [inline docs/Contribute/OnlineEditor_Edit.md]: "http" -> "https" (I didn't know http didn't work on this url, just fixed it now)

File: docs/Mods/ExtraUtilities2/CustomMachines/IMachine.md [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+
+- `getInputSlots()`: Returns all input slots as a List of [IMachineSlot](/Mods/ExtraUtilities2/CustomMachines/IMachineSlot).
+- `getOutputSlots()`: Returns all output slots as a List of [IMachineSlot](/Mods/ExtraUtilities2/CustomMachines/IMachineSlot).
+- `getSlot()`: Returns the [IMachineSlot](/Mods/ExtraUtilities2/CustomMachines/IMachineSlot) matching the name.
+
+
+## Naming the machine
+So far, all our machines will be named `machine.crafttweaker:your_machine_name` where `your_machine_name` is whatever name you used to create the machine.
+
+If you want the machine name localized, use either CrT's [IGame](/Vanilla/Game/IGame) capabilities or a custom lang file, though latter is preferred since there are some issues with CrT's way of doing the localization.
```
Comment (jaredlll08) [inline docs/Mods/ExtraUtilities2/CustomMachines/IMachine.md]: No longer any issues I don't believe ;p

Comment (kindlich) [reply docs/Mods/ExtraUtilities2/CustomMachines/IMachine.md]: Added an example how to use either the CrT IGame method or a lang file

**Pull request merged**